rest/handler/prometheushandler.go

```diff
@@ -12,14 +13,10 @@ import (
 const serverNamespace = "http_server"
 
 var (
-	metricServerReqDur = metric.NewHistogramVec(&metric.HistogramVecOpts{
-		Namespace: serverNamespace,
-		Subsystem: "requests",
-		Name:      "duration_ms",
-		Help:      "http server requests duration(ms).",
-		Labels:    []string{"path", "method", "code"},
-		Buckets:   []float64{5, 10, 25, 50, 100, 250, 500, 750, 1000},
-	})
+	serverReqDurBuckets    = []float64{5, 10, 25, 50, 100, 250, 500, 750, 1000}
```

Comment:
Many users might not configure this value, so the default value could be set to cover a wider range.

Reply:
OK, let's discuss internally whether to adjust the default value.